src/pages/index.js

```diff
@@ -3,22 +3,101 @@ import {
   initialCards,
   config,
   profileEditButton,
+  profileEditAvatarButton,
   addNewCardButton,
   profileTitleInput,
   profileDescriptionInput,
   addNewCardFormEl,
   profileEditFormEl,
+  profileEditAvatarFormEl,
 } from "../utils/constants.js";
 import PopupWithForm from "../components/PopupWithForm.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import FormValidator from "../components/FormValidator.js";
 import UserInfo from "../components/UserInfo.js";
 import Card from "../components/Card.js";
 import Section from "../components/Section.js";
+import Api from "../components/Api.js";
 
 const pageUserInfo = new UserInfo({
   nameSelector: ".profile__title",
   jobSelector: ".profile__description",
+  avatarSelector: "#profile-avatar",
+});
+
+const api = new Api({
+  baseURL: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "98cf51ba-481b-4eec-9343-23efbe8fbf82",
+    "Content-Type": "application/json",
+  },
+});
+
+let user = api.getUserInfo().catch((err) => console.log(err));
+let apiCards = api.getInitialCards().catch((err) => console.log(err));
+let cardSection;
+
+function createCard(cardData) {
+  const card = new Card(
+    cardData,
+    "#card-template",
+    (cardData) => {
+      imageViewerPopup.open(cardData);
+    },
+    () => {
+      deleteCardPopup.open();
+      deleteCardPopup.setSubmitAction(() => {
+        deleteCardPopup.setLoadingState(true);
+        api
+          .deleteCard(cardData._id)
+          .then(() => {
+            deleteCardPopup.close();
+            card.remove();
+          })
+          .catch((err) => console.log(err))
+          .finally(deleteCardPopup.setLoadingState(false));
```

Comment:
Needs correction: `finally` method accepts a function `.finally(() => deleteCardPopup.setLoadingState(false));`

Reply:
corrected. thank you.